xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1500,18 +1509,24 @@ The current depth exceeds the recursive limit set by the maximum depth.</excepti
         <ReturnType>System.Buffers.ReadOnlySequence&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the value of the last processed token as a <see cref="T:System.ReadOnlySpan`1" /> slice of the input payload. If the JSON is provided within a <see cref="T:System.Buffers.ReadOnlySequence`1" /> and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
+        <summary>Gets the raw value of the last processed token as a ReadOnlySequence&lt;byte&gt; slice of the input payload, only if the token is contained within multiple segments.</summary>
         <value>A byte read-only sequence.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
+If the JSON is provided within a ReadOnlySequence&lt;byte&gt; and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, `ValueSequence` contains the token value.
```

Comment:
```suggestion
If the JSON is provided within a ReadOnlySequence&lt;byte&gt; and the slice that represents the token value fits in a single segment, then <xref:System.Text.Json.Utf8JsonReader.ValueSpan> contains the sliced value since it can be represented as a span. Otherwise, `ValueSequence` contains the token value.
```